sfge/src/main/java/com/salesforce/config/UserFacingMessages.java

```diff
@@ -0,0 +1,31 @@
+package com.salesforce.config;
+
+/**
+ * Contains error message constants that will be displayed to users. TODO: move all other
+ * user-facing messages here.
+ */
+public final class UserFacingMessages {
+
+    /** UserActionException * */
+
+    // format: filename,defined type, line number
+    public static final String UNREACHABLE_CODE =
+            "Please remove unreachable code to proceed with analysis: %s,%s:%d";
+
+    /** CRUD/FLS Violation messages * */
```

Comment:
@teresa-allen-sfdc These messages could use your magic. These have existed for a while but putting them in one place for easier review.